src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,51 @@
+package duke;
+
+import java.util.Scanner;
+
+
+
+/**
+ * Stores the scanner and handles what information to display to the user.
+ */
+public class Ui {
+
+    private final Scanner sc;
+
+    /**
+     * Constructor for starting the Ui and launching Stage for JavaFX
+     */
+    public Ui() {
+        greet();
+        sc = new Scanner(System.in);
+    }
+
+    public boolean hasUserInput() {
+        return sc.hasNextLine();
+    }
+
+    public String userInput() {
```

Comment:
Perhaps consider renaming this method to start with a verb so that it complies with the coding standard? For example, a possible alternative could be `getUserInput()`.